automation/terraform/modules/o1-testnet/testnet.tf

```diff
@@ -52,65 +52,67 @@ module "kubernetes_testnet" {
 
   additional_peers = [for peer in local.static_peers : peer.full_peer]
   runtime_config   = var.use_embedded_runtime_config ? "" : data.local_file.genesis_ledger.content
-  
+
   seed_zone   = var.seed_zone
   seed_region = var.seed_region
 
   archive_configs = local.archive_node_configs
 
-  mina_archive_schema = var.mina_archive_schema
+  mina_archive_schema           = var.mina_archive_schema
   mina_archive_schema_aux_files = var.mina_archive_schema_aux_files
 
   snark_coordinators = var.snark_coordinators
 
-  block_producer_key_pass = var.block_producer_key_pass
-  block_producer_configs = [for i, bp in local.block_producer_configs:
+  # block_producer_key_pass = var.block_producer_key_pass
+  block_producer_configs = [for i, bp in local.block_producer_configs :
     {
-      name                   = bp.name
-      class                  = bp.class
+      name  = bp.name
```

Comment:
Did you ran some formatter tool or manually changed tabs? It looks like there is some inconsistency in tabbing :)